app/views/games/show.html.erb

```diff
@@ -1 +1,49 @@
-<h1><%= @game.name %></h1>
\ No newline at end of file
+
+<%= javascript_include_tag "board" %>
+<div class="col-11 offset 0.5 container">
+  <div class="col-8 offset-2">
+    <h1 class="board-title"><%= @game.name %></h1>
+    <br />
+  </div>
+      
+  <div class="row">
+    <div class="col-2 my sidebar">
+      <%= @game.user.email %>
+    </div>
+    <div class="col-8">
+      <div class="board">
+        
+        <% row = 8 %>
+        <% while row > 0 %>
+          <% for column in 1..8 do %>
+
+            <% if row % 2 == 0 && column % 2 == 0 %>
```

Comment:
@nerdycat99 one last thing I would add:

combining the conditions for black and putting them into a helper method so you end up with sth like:
 <% black_cell(row, column) ? colour = 'black' : colour = 'white' %>